apps/angular/src/core/services/anime.service.ts

```diff
@@ -0,0 +1,28 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable, inject } from '@angular/core';
+import { Observable, map } from 'rxjs';
+
+import { AnimeDto } from '@js-camp/core/dtos/anime.dto';
+import { AnimeDomain } from '@js-camp/core/models/anime-domain.model';
+
+import { AnimeMapper } from '@js-camp/core/mappers/anime-mapper.service';
+
+/** HTTPS requests. */
+@Injectable({
+	providedIn: 'root',
+})
+export class AnimeService {
+	private http: HttpClient = inject(HttpClient);
+
+	private animeMapper: AnimeMapper = inject(AnimeMapper);
+
+	/** Get JSON from API. */
+	public getAnime(): Observable<AnimeDomain[]> {
```

Comment:
[Avoid context duplication](https://wiki.saritasa.rocks/frontend/languages/ts-js/naming#avoid-context-duplication), as anime doesn't have plural, we could use `list`.
```suggestion
	public getList(): Observable<AnimeDomain[]> {
```